src/server/services/hli/status-line-items/generic.ts

```diff
@@ -19,6 +19,22 @@ function getNamenBetrokkenen(
   return `${names.join(', ')} en ${lastName}`;
 }
 
+export function getBetrokkenKinderen(
+  regeling: ZorgnedAanvraagWithRelatedPersonsTransformed
+) {
+  const names = regeling.betrokkenPersonen
+    .filter((person) => !!person.name)
+    .map(
+      (person) =>
+        `${person.name}${person.dateOfBirthFormatted ? ` (geboren op ${person.dateOfBirthFormatted})` : ''}`
+    );
+  if (names.length <= 1) {
```

Comment:
bedoel je niet `>=` of `===` ?

```suggestion
  if (names.length >= 1) {
```

```suggestion
  if (names.length == 1) {
```

Reply:
Nee want als er geen of één naam hebben willen we geen separator hebben en wel een string teruggeven.